README.md

```diff
@@ -91,6 +91,11 @@ if you are interested to contribute to this repo.
 
 Please submit an issue at: [Issues](https://github.com/ceph/ceph-csi/issues)
 
+## Weekly Bug Traige call
```

Comment:
Oops, missed this,
s/Traige/Triage/

Reply:
@ajarr this is what happens when you attend a meeting and send PR at same time :).. Thanks